cadence/contracts/LendingComptroller.cdc

```diff
@@ -260,7 +258,7 @@ pub contract LendingComptroller {
         /// @Param borrowUnderlyingAmountScaled - Borrow amount scaled by 1e18
         /// @Return error code
         ///
-        pub fun borrowAllowed(
+        access(all) fun borrowAllowed(
```

Comment:
`view`?

Reply:
Cannot be made view as non-view operation performed in `callerAllowed()`